sound/soc/sof/Kconfig

```diff
@@ -195,6 +195,16 @@ config SND_SOC_SOF_DEBUG_IPC_FLOOD_TEST
 	  Say Y if you want to enable IPC flood test.
 	  If unsure, select "N".
 
+config SND_SOC_SOF_DEBUG_IPC_FLOOD_TEST_CLIENT
+	tristate "SOF enable IPC flood test client"
```

Comment:
I would rather merge this and the old IPC flood removal patch and use the same Kconfig.
Breaking savedefconfig or CI with renamed option for the same functionality has no benefit to users or developers.

Reply:
Hrm, it is going to break userspace... With the new flood test we place the files under a directory no like it was previously when the files were created directly under sof_debugfs_root.

Reply:
@plbossart , @ranj063 , @fredoh9 , something like this:
https://github.com/ujfalusi/sof-linux/commit/d3455cc8c86d86e3dc5727a0a39d598cdd262be1
With this commit the Kconfig remains the same and we will have the legacy debugfs files created as a link to the first flood auxdev instance.

The commit message needs to be updated and I think I will split the patch up also to first create only one instance of flood test, then add the Kconfig parameter to configure the number of concurrent clients.

Most likely the Kconfig also need a second/third look.

I still have my doubts about the validity of auxbus for ipc_flood and the probe_test later on.
The first commit needs to be revisited as well, I have a change which is no longer needed there.

Reply:
It looks like this on real life:
```
ls -al /sys/kernel/debug/sof/
total 0
drwxr-xr-x  4 root root 0  9. 4. 10:21 .
drwx------ 32 root root 0  9. 4. 10:21 ..
-r--r--r--  1 root root 0  9. 4. 10:21 cc_version
-r--r--r--  1 root root 0  9. 4. 10:21 debug
-r--r--r--  1 root root 0  9. 4. 10:21 dsp
-r--r--r--  1 root root 0  9. 4. 10:21 etrace
-r--r--r--  1 root root 0  9. 4. 10:21 exception
--w-------  1 root root 0  9. 4. 10:21 filter
-r--r--r--  1 root root 0  9. 4. 10:21 fw_version
-r--r--r--  1 root root 0  9. 4. 10:21 hda
-r--r--r--  1 root root 0  9. 4. 10:21 inbox
lrwxrwxrwx  1 root root 0  9. 4. 10:21 ipc_flood_count -> snd_sof.ipc_test.0/ipc_flood_count
lrwxrwxrwx  1 root root 0  9. 4. 10:21 ipc_flood_duration_ms -> snd_sof.ipc_test.0/ipc_flood_duration_ms
-r--r--r--  1 root root 0  9. 4. 10:21 outbox
-r--r--r--  1 root root 0  9. 4. 10:21 pp
-r--r--r--  1 root root 0  9. 4. 10:21 regs
drwxr-xr-x  2 root root 0  9. 4. 10:21 snd_sof.ipc_test.0
drwxr-xr-x  2 root root 0  9. 4. 10:21 snd_sof.ipc_test.1
-r--r--r--  1 root root 0  9. 4. 10:21 stream
-r--r--r--  1 root root 0  9. 4. 10:21 trace
```